### PR TITLE
Responsive UI km

### LINKED ISSUE #88: Responsiveness: some elements squish awkwardly

### PR DESCRIPTION
**Pull request by Kristiannmiller**
# Pull Request

## What's this PR do? 

- [ ] New feature (non-breaking change which adds functionality)
- [X] Bug fix 
- [ ] Testing
- [X] Styling
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Description of changes?
- Fixes syntax bug in styled components
- Addresses date related bug (temporary. Basically, it just adds a day to the date. After researching i found that this might be a common issue with Date and dayjs, and I'll continue to look into it for a more responsive fix).
- Adds responsive design and new styled components for easy transition between web, tablet, and mobile devices.

## Any background context you want to provide? 
- Some of the responsive design might take a little more work and polish
- The image for each wedding needs to be 1:1 in order ti fit correctly. Ideally, we should add some kind of cropping service to the new wedding form, or maybe just a prompt that says ideal image should be a square. 

## What are the relevant tickets? 
* Addresses #105 
* Closes #88

### CODE REVIEW

**Review by ericberglund117 — APPROVED** [review]
The responsiveness and styled components look amazing! Thank you for putting in the extra effort to have that ready by code fair.

File: src/components/AddWeddingForm/AddWeddingForm.css [hunk 1]
```diff
@@ -1,8 +1,49 @@
...
+  padding-top: 5%;
+}
+#weddingFormInput {
+  transition: all .5s ease-Out;
+}
+#weddingFormTitle {
+  transition: all .5s ease-Out;
+  font-size: 4vw;
+}
+@media screen and (max-width: 1000px) {
```
Comment (hannakim91) [inline src/components/AddWeddingForm/AddWeddingForm.css:18]: Thanks for spending time on making this responsive!

File: src/components/AddWeddingForm/AddWeddingForm.css [hunk 2]
```diff
@@ -1,8 +1,49 @@
...
   width: 100%;
   height: 100%;
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
+  padding-top: 5%;
+}
+#weddingFormInput {
+  transition: all .5s ease-Out;
```
Comment (hannakim91) [inline src/components/AddWeddingForm/AddWeddingForm.css:12]: This is a nice touch for better user feedback with the timed transition!

File: src/components/App/App.css [hunk 3]
```diff
@@ -46,10 +46,22 @@ footer, header, hgroup, menu, nav, section {
   background-color: #f5f5f5;
   position: sticky;
   top: 0;
-  z-index: 1;
+  z-index: 500;
```
Comment (hannakim91) [inline src/components/App/App.css:49]: Cool to see this in action after only having read about it

File: src/components/App/styledComponents.styles.ts [hunk 4]
```diff
@@ -26,11 +116,25 @@ export const StyledCard = styled.section<StyledCardProps>`
   height: ${props => {
     if(props.contents === 'wedding') {
       return '40em;'
+    } else if(props.contents === 'empty') {
```
Comment (hannakim91) [inline src/components/App/styledComponents.styles.ts:119]: I didn't realize you could put this level of specific conditional rendering for styled components!

**Review by hannakim91 — APPROVED** [review]
Awesome work on all the additional styling!

**Pull request merged**